android/app/build.gradle

```diff
@@ -203,6 +206,14 @@ android {
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
             signingConfig signingConfigs.release
         }
+        pr {
+            initWith release
+            applicationIdSuffix ".pr"
+            versionNameSuffix ".pr"
+            matchingFallbacks = ["release"]
+            // necessary to make status-go config generation work
```

Comment:
rather "necessary to make react-native-config's code generation work". These configs are used not only for status-go

Reply:
Fixed.